guides/common/modules/proc_installing-flatpak-applications-on-project-hosts.adoc

```diff
@@ -22,12 +22,21 @@ The token generates automatically when you run `podman login`.
 ----
 $ flatpak remote-add --authenticator-name=org.flatpak.Authenticator.Oci katello oci+https://_{foreman-example-com}_/
 ----
-. Log in using Podman:
+. Configure the host to use container certificates for authentication.  
+Authentication with certificates allows {Project} or {SmartProxy} to restrict the access of the host to container and Flatpak repositories based on the assigned content view and lifecycle environment.
+
+* When registering a host to {Project} or {SmartProxy}, select the *Set up container registry certs* checkbox.
+* Alternatively, if you want to configure certificate authentication for container and flatpak content on a registered host:
+- Run the *Flatpak - Login to registry via podman* job template on the host.
+- Set the *Set up certificate authentication* input to `true` and enter the URL of the {Project} or {SmartProxy} server as the registry URL.
+
+. Alternatively, you can log in using Podman:
```

Comment:
I think this should be a single step whose point is to login the host to the registry. Then you have alternatives:
 1. Set up auth with certs and login via REX.
 2. Login manually.

Please, rework.

The same below.